src/serpentTools/parsers/depmatrix.py

```diff
@@ -41,6 +41,8 @@ class DepmtxReader(BaseReader, SparseReaderMixin):
     ----------
     deltaT: float
         Length of depletion interval
+    flx: float
```

Comment:
Would prefer to have the full variable name just to be more explicit. And it should be typed as optional float, right? Since for older versions of serpent that don't output this data, users should expect it to not always be a float.